druid/examples/game_of_life.rs

```diff
@@ -0,0 +1,171 @@
+// Copyright 2019 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Game of life
+
+use druid::kurbo::BezPath;
+use druid::piet::{FontBuilder, ImageFormat, InterpolationMode, Text, TextLayoutBuilder};
+
+
+use druid::{Affine, AppLauncher, BoxConstraints, Color, Env, Event, EventCtx, LayoutCtx, LifeCycle, LifeCycleCtx, LocalizedString, PaintCtx, Point, Rect, RenderContext, Size, UpdateCtx, Widget, WindowDesc, Data};
+
+
+const POOL_SIZE: i32 = 1600;
```

Comment:
I think this should be a `usize`, which is the type used for most stdlib indexes?

Reply:
Also: I think the const can be used in the array declaration. Since I think you mean them to match, best to use the const.
Secondly, if you intend a square, and want to make your life 'easier', I would define a 'SIDE_LENGTH' const, and make this 'SIDE_LENGTH * SIDE_LENGTH'. (SIDE_LENGTH = 40, here)